userapi/src/main/java/fk/prof/userapi/Configuration.java

```diff
@@ -0,0 +1,107 @@
+package fk.prof.userapi;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.vertx.core.DeploymentOptions;
+import io.vertx.core.json.JsonObject;
+
+import javax.validation.Valid;
+import javax.validation.constraints.NotNull;
+import java.util.Map;
+
+/**
+ * Created by gaurav.ashok on 15/05/17.
+ */
+public class Configuration {
+
+  @JsonProperty("profile.retention.duration.min")
+  public Integer profileRetentionDurationMin = 30;
```

Comment:
Members should not be public. Should only expose a getter. Configuration should ideally be immutable once generated from config file and validated